index.d.ts

```diff
@@ -1,10 +1,10 @@
 declare module 'react-native-geocoding' {
     export default class Geocoder {
-        init(apiKey: string, options: Object): void;
-        isInit(): boolean;
-        setApiKey(API_KEY: string): void;
-        from(...params: any[]): Promise<void>;
```

Comment:
from doesn't return void, it returns a google geocoding response like so
https://developers.google.com/maps/documentation/geocoding/overview#GeocodingResponses or it throws an error.

What I did locally was create an interface that contains the geocoding response like so
`
interface geocodingResponse {
        plus_code: {compound_code: string, global_code: string}
        results: Array;
        status: string;
    }
`

and changed the declartion to:
`static from(...params: any[]): Promise<geocodingResponse>;
`

na